### PR TITLE
Helpful Message After Generator Install

### PR DESCRIPTION
**Pull request by aaronvb**
- Added a helpful message after base generator install.
- Added a helpful message after react with redux generator install.
- Added tests for both messages.

`rails generate react_on_rails:install`

screen shot 2016-01-14 at 1 35 21 pm

`rails generate react_on_rails:install -R`

screen shot 2016-01-14 at 1 37 04 pm

Resloves #184

### CODE REVIEW

**Comment by justin808** [comment]
@aaronvb This looks awesome. Could you change "node-server-hot" to "express-server". I think that's a better name for the webpack dev server.

@alexfedoseev @robwise Agree?

Since there's no "cold", non-hot-reloading version of express, I think just `npm run express-server` is better. Or maybe `npm run wds` for "webpack dev server"?

**Comment by alexfedoseev** [comment]
`npm run express-server` :+1:

File: lib/generators/react_on_rails/templates/base/base/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "postinstall": "cd client && npm install",
     "test": "rspec && (cd client && npm run lint)",
-    "node-server-hot": "echo 'visit http://localhost:4000' && cd client && npm start",
+    "express-server": "echo 'visit http://localhost:4000' && cd client && npm start",
```
Comment (aaronvb) [inline lib/generators/react_on_rails/templates/base/base/package.json:13]: Updated the npm run command `node-server-hot` to `express-server`.

**Comment by aaronvb** [comment]
The updated helpful message with `npm run express-server`

screen shot 2016-01-17 at 9 59 15 pm

**Comment by justin808** [comment]
Thanks @aaronvb. Is this ready for merge and a dot release?

**Comment by aaronvb** [comment]
@justin808 I just pushed an edit to the `docs/tutorial-v2.md` to reflect the `npm run express-server` change. Once the CI passes it will be good to go.

**Pull request merged**